packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/mira/DKG.java

```diff
@@ -19,6 +23,12 @@ public class DKG {
 	public static final String NAME = "name:string";
 	public static final String DESCRIPTION = "description:string";
 	public static final String EMBEDDINGS = "description:dense_vector";
+	public static final String GEONAMES = "geonames";
+
+	public DKG(String curie) {
+		this.curie = curie;
+		locations = new ArrayList<>();
+	}
```

Comment:
I am not sure how this is modelled in MIRA, but the last time I worked with it on the WM project, it isn't such that "a curie has a list of geonames", but rather "geoname is a type of curie with id/description/label ...etc"

Reply:
The proposal from @liunelson has the geonames as a list of locations on the grounding - I'm under the impression that this is just for our own internal representation?